cfme/services/myservice/__init__.py

```diff
@@ -34,6 +35,19 @@ def __init__(self, appliance, name=None, description=None, vm_name=None):
         self.vm_name = vm_name
         self.parent = self.appliance.context
 
+    @property
+    def rest_obj(self):
```

Comment:
I think it would be better to be consistent with `rest_api_entity`, as this property name is already used for a few classes and is consistent with use of the word `entity`, though this is not _yet_ `BaseEntity` derived.

Reply:
Check out what @valaparthvi is doing in #9329 !